docker-compose.override.yml

```diff
@@ -17,3 +17,55 @@ services:
         - perma_web/package.json
         - perma_web/perma-warcprox-ca.pem
         - perma_web/lil-archive-keyring.gpg
+
+  #
+  # Perma Payments
+  #
+  pp_db:
+    image: registry.lil.tools/library/postgres:12.8
+    environment:
+      - POSTGRES_PASSWORD=example
+    volumes:
+      - pp_db_data:/var/lib/postgresql/data:delegated
+    networks:
+      - perma_payments
+
+  perma-payments:
+    image: registry.lil.tools/harvardlil/perma-payments:0.37
+    # hack: sleep to give the database time to start up
+    command: >
+      sh -c "sleep 5 && ./manage.py migrate && fab run"
+    ports:
+      - "127.0.0.1:80:80"
+    depends_on:
+      - pp_db
+    environment:
+      - DOCKERIZED=True
+      - DJANGO__DATABASES__default__HOST=pp_db
+    # TO CUSTOMIZE CONFIG:
+    # copy ./services/docker/perma-payments/settings.py.example
+    # to ./services/docker/perma-payments/settings.py,
+    # alter config as desired, in the usual way, then comment in this volume
+    # and re-run docker-compose up
+    # volumes:
+    #   - ./services/docker/perma-payments/settings.py:/perma-payments/config/settings/settings.py
+    networks:
+      - default
+      - perma_payments
+
+networks:
+  default:
+  # perma_payments:
+      # name: perma-payments_default
```

Comment:
Interesting to see this is commented out!

Have you tested the normal setup, where Perma Payments is not running externally, and Perma's containers have been starter with plain old `docker compose up -d`? If yes, and if that works, then I bet that means the entire `network` stanza can be dropped here.

Reply:
Yes, I have. I thought the default would need to be set. I'll see what happens without it.

Reply:
I think I'm going to leave the webrecorder piece and have it and the networks just commented out for people to add according to their preference.

Reply:
Okay! To me, having it present in `docker-compose.yml` is sufficient, but if you think having something in the override file too is better, cool cool!

Reply:
I think you know better than I how people actually use this, so I am removing.